lib/potluck.rb

```diff
@@ -0,0 +1,35 @@
+class Potluck
+  attr_reader :date,
+              :dishes
+
+  def initialize(date)
+    @date = date
+    @dishes = []
+    @menu_items = Hash.new([0])
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def get_all_from_category(category)
+    @dishes.find_all do |dish|
+      if dish.category == category
```

Comment:
You can refactor this to remove the if statement and just have dish.category == category